drivers/i2c/i2c_npcx_controller.c

```diff
@@ -414,6 +414,9 @@ static int i2c_ctrl_recovery(const struct device *dev)
 	struct i2c_ctrl_data *const data = DRV_DATA(dev);
 	int ret;
 
+	if (data->oper_state != NPCX_I2C_ERROR_RECOVERY)
```

Comment:
braces around single statement body

Reply:
Thanks. I will remind myself of this coding rule.

Reply:
Done.